cgmanifest.json

```diff
@@ -840,13 +860,23 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "CUnit",
+          "version": "2.1.3",
+          "downloadUrl": "https://downloads.sourceforge.net/CUnit/CUnit-2.1-3.tar.bz2"
+        }
+      }
+    },
     {
       "component": {
         "type": "other",
         "other": {
           "name": "Cython",
-          "version": "0.28.5",
-          "downloadUrl": "https://github.com/cython/cython/archive/0.28.5.tar.gz"
+          "version": "0.29.13",
+          "downloadUrl": "https://github.com/cython/cython/archive/0.29.13.tar.gz"
```

Comment:
Revert this change to cgmanifest since this PR doesn't update Cython any more.

Reply:
Done